### PR TITLE
Makes the surgery computer useful

### PR DESCRIPTION
**Pull request by pinatacolada**
Adds nanoUI to it, and makes the surgery computer announce the patient's
health and beep when it degrades past certain thresholds. Also lets you
customize those tresholds and mute them individually if you want to.

exhibit A

exhibit B

Exhibit C

This is my first PR and I did it more so I could get familiar with byond
and git, please tell if something's wrong

### CODE REVIEW

**Comment by NTSAM** [comment]
https://www.youtube.com/watch?v=QXCyzPMEoXM

File: code/game/machinery/computer/Operating.dm [hunk 1]
```diff
@@ -82,9 +159,43 @@
...
+				sleep(10)
+				atom_say("[src.victim.real_name], [src.victim.b_type] blood, [src.victim.stat ? "Non-Responsive" : "Awake"]")
+				atom_say("[round(src.victim.health)]")
+			if(src.victim.health <= -50 && crit)
+				playsound(src.loc, 'sound/machines/defib_success.ogg', 50, 0)
+			if(src.victim.getOxyLoss()>oxyAlarm)
+				playsound(src.loc, 'sound/machines/defib_saftyOff.ogg', 50, 0)
+			if(healthAnnounce)
+				atom_say("[round(src.victim.health)]")
+			sleep(60)
```
Comment (Fox-McCloud) [inline code/game/machinery/computer/Operating.dm]: ...a sleep...in processing.

NO NO NO NO NO NO NO NO NO NO NO

Comment (pinatacolada) [inline code/game/machinery/computer/Operating.dm]: what's the alternative? i just need the thing to wait so it doesn't spam the health every tick

Comment (Fox-McCloud) [inline code/game/machinery/computer/Operating.dm]: you can create a timer whereby when stuff is announced it sets your timer's var to world.time....what you'd then do is have it not announce things if world.time+<x> have passed.

Comment (pinatacolada) [inline code/game/machinery/computer/Operating.dm]: i'll do this tomorrow then

**Comment by Fox-McCloud** [comment]
I'll talk to @Aurorablade about her plans for surgery computers--I know there was something involving surgery and steps and what comes next involving these computers---though I don't know the specifics; depending on how that works, that would probably be more appropriate than this.

File: code/game/machinery/computer/Operating.dm (hunk 1 above)
Comment (Fox-McCloud) [inline code/game/machinery/computer/Operating.dm]: also, another thing is the src. isn't needed in these case, as it's already implied.

**Comment by CupcakesNHacksaws** [comment]
Would it be possible to have the computer show you the steps for the surgery needed?

**Comment by AceMcLazer** [comment]
@CupcakesNHacksaws That would kinda Destroy the Purpose of surgery being INTENTIONALLY difficult then would it not? Make every grey tide with a Glass shard and a Surgery board a surgeon

**Comment by tigercat2000** [comment]
@AceMcLazer given that they are documented on the wiki for anyone to look at it's probably not a big deal

but bay surgery is shit so I doubt you could do "what they need" or even "what to do for each surgery"

**Comment by Aurorablade** [comment]
yeah what i am doing with my PR is just showing of next surgery step and thats it. I can adapt to any UI changes as needed.

**Comment by johannhawk** [comment]
all this needs now is to play the trauma center/team OST in the background...

**Comment by pinatacolada** [comment]
updated

**Comment by FalseIncarnate** [comment]
:+1: for a step towards making this computer actually worth the tile it occupies.

Only thing I'd consider is a second thing like the oxy threshold setting, except for overall health.

Having it announce their health sounds handy, but would be annoying to have to switch on and off because one patient is critical and the next is at 100% but is in for something like implant removal or such.

**Comment by Fox-McCloud** [comment]
@Markolie 

This look alright to you? UI's aren't DZD and I's strong points.

**Comment by Markolie** [comment]
Looks good to me!

**Pull request merged**